### DISCUSSION TITLE
Option -g extras

### DISCUSSION THREAD

**Discussion by felipelsantos** [prowler-cloud/prowler#1746] [General]
Hello, 

Would be interesting to return the option -g "extras" to checking just different rules from CIS, AWS, and so on.

Thanks,

**Comment by toniblyx** [comment]
Hi @felipelsantos we don´t have such option in v3 because extras no longer exist as a group. You can use a space separate list of checks you want to execute or even create a json file with those you want to run ike specified here https://docs.prowler.cloud/en/latest/tutorials/misc/#checks. For compliance specific stuff see here https://docs.prowler.cloud/en/latest/tutorials/compliance/

**Reply by EspressoTimez** [reply]
Hi @toniblyx are we not able to use version 2, where the '-g' argument was still in use?

**Reply by toniblyx** [reply]
the option `-g group` is available in Prowler v2 as that still available here `https://github.com/prowler-cloud/prowler/tree/prowler-2` however such an option is not available in v3 but you can do the same thing in a different ways.
1. Create your own list of checks to execute and run prowler for that like here https://docs.prowler.cloud/en/latest/tutorials/misc/#checks `prowler <provider> -C/--checks-file <checks_list>.json`
2. Create a category called `extras` (editing all needed checks and addign extras in metadata.json files per check in categories section and sending us a pull request) using this mapping https://docs.prowler.cloud/en/latest/tutorials/aws/v2_to_v3_checks_mapping/ and https://github.com/prowler-cloud/prowler/blob/prowler-2/groups/group7_extras#L17 once that is done you can execute prowler like `prowler <provider> --categories extras`
I hope that helps

**Reply by EspressoTimez** [reply]
thank you for your lengthy response it was helpful, however, I'm struggling with the first method:

`C:\Users\Documents>prowler aws -C/--checks-file checks.json`
`usage: prowler [-h] [-v] {aws,azure} ...`
`prowler: error: unrecognized arguments: checks.json`

For your first method in version 3, am I just formatting it incorrectly?
I did create the file as well and store it in the local folder.

**Reply by toniblyx** [reply]
when we specify ` C/--checks-file` in commands  is because you have to use either `-C` or `--checks-file`. So in this case you have to follow instructions in documentation to create your own check-list-file.json and once done run `prowler aws --checks-file check-list-file.json`. Those are example, you can use your own file names.
